### ISSUE TITLE
ERROR EXPORTING TO TF SAVED_MODEL

### ISSUE THREAD

**Issue by ilantrup** [ultralytics/ultralytics#1851]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi! I have trained my custom model with YOLOv8x and saved it on my own computer. When I try to export the .pt model to TF saved_model it throws me this error:
Ultralytics YOLOv8.0.58  Python-3.10.9 torch-2.0.0+cpu CPU
Model summary (fused): 268 layers, 68201571 parameters, 0 gradients, 257.8 GFLOPs

PyTorch: starting from best.pt with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 85, 8400) (130.5 MB)

TensorFlow SavedModel: starting export with tensorflow 2.12.0...

ONNX: starting export with onnx 1.13.1...
============== Diagnostic Run torch.onnx.export version 2.0.0+cpu ==============
verbose: False, log level: Level.ERROR
======================= 0 NONE 0 NOTE 0 WARNING 0 ERROR ========================

ONNX: simplifying with onnxsim 0.4.19...
ONNX: export success  11.6s, saved as best.onnx (0.9 MB)

TensorFlow SavedModel: running 'onnx2tf -i best.onnx -o best_saved_model -nuo --non_verbose'
TensorFlow SavedModel: export failure  19.0s: SavedModel file does not exist at: best_saved_model\{saved_model.pbtxt|saved_model.pb}

I am executing this code: 
from ultralytics import YOLO
modelo = YOLO('best.pt')
modelo.export(format='saved_model', dynamic=True)

Please someone could help me. 
Thanks!

**Comment by glenn-jocher** [comment]
@charlespunter, can you please check that the directory where you are trying to save the Tensorflow SavedModel exists? Based on the error message, it seems like the specified directory `best_saved_model` does not exist. 

Alternatively, have you tried using the `--saved_model` argument to explicitly specify the output directory? Example: `modelo.export(format='saved_model', dynamic=True, saved_model='path/to/saved_model')`.

**Comment by zperezedgar** [comment]
> @charlespunter, can you please check that the directory where you are trying to save the Tensorflow SavedModel exists? Based on the error message, it seems like the specified directory `best_saved_model` does not exist.
> 
> Alternatively, have you tried using the `--saved_model` argument to explicitly specify the output directory? Example: `modelo.export(format='saved_model', dynamic=True, saved_model='path/to/saved_model')`.

I have the same issue when trying to export a model to tflite from Google Colab. Maybe the problem is that the creation of the directory in gdrive has a delay. Then, when the script looks for that directory it has not been created yet?
